test/e2e/common/storage/configmap_volume.go

```diff
@@ -61,7 +59,7 @@ var _ = SIGDescribe("ConfigMap", func() {
 		doConfigMapE2EWithoutMappings(ctx, f, false, 0, &defaultMode)
 	})
 
-	f.It("should be consumable from pods in volume as non-root with defaultMode and fsGroup set [LinuxOnly]", nodefeature.FSGroup, feature.FSGroup, func(ctx context.Context) {
+	f.It("should be consumable from pods in volume as non-root with defaultMode and fsGroup set [LinuxOnly]", func(ctx context.Context) {
```

Comment:
This test and several others started running in https://testgrid.k8s.io/sig-release-master-blocking#kind-master-parallel between  4766d191f..32130691a, which is exactly this commit.

Works as intended! Thanks.